docs/user-guide/configure.md

```diff
@@ -172,6 +172,23 @@ Experimental feature: some rules support message arguments. For example, when co
 
 With formats that don't support a function like JSON, you can use a `printf`-like format (e.g., `%s`). On the other hand, with JS format, you can use both a `printf`-like format and a function.
 
+### `url`
+
+You can use the `url` secondary option to provide a custom link to external docs. These urls can then be displayed in custom formatters.
+
+For examplel
+
+```json
+{
+  "color-no-hex": [
+    true,
+    {
+      "url": "https://stylelint.io"
+    }
+  ]
+}
+```
```

Comment:
[suggest] Thanks, the added description sounds good! Additionally, what about the following points?

- Fix a typo in "For example".
- Save a few lines in the code example.
- Use another domain other than `stylelint.io` to clarify the purpose of this option.

````suggestion
For example:

```json
{
  "color-no-hex": [true, {"url": "https://example.org/your-custom-doc"}]
}
```
````

Reply:
Boy, embarrassing typo there. Done.